ttnn/cpp/ttnn/tensor/layout/page_config.hpp

```diff
@@ -0,0 +1,68 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <array>
+
+#include "ttnn/tensor/types.hpp"
+#include "ttnn/tensor/enum_types.hpp"
+
+#include "impl/tile/tile.hpp"
+
+#include "alignment.hpp"
+#include "size.hpp"
+
+#include <optional>
+
+namespace tt::tt_metal {
+
+class RowMajorPageConfig {
```

Comment:
All of these are good candidates to move into tt-metal or incrementally if it's easier

Reply:
True. PageConfig might be removed as we evolve, but in general - yes

Reply:
Agreed